src/apps/ENERGY-OMPTarget.cpp

```diff
@@ -161,35 +161,39 @@ void ENERGY::runOpenMPTargetVariant(VariantID vid)
     startTimer();
     for (RepIndex_type irep = 0; irep < run_reps; ++irep) {
 
-      RAJA::forall<RAJA::omp_target_parallel_for_exec<NUMTEAMS>>(
-        RAJA::RangeSegment(ibegin, iend), [=](int i) {
-        ENERGY_BODY1;
-      });
-
-      RAJA::forall<RAJA::omp_target_parallel_for_exec<NUMTEAMS>>(
-        RAJA::RangeSegment(ibegin, iend), [=](int i) {
-        ENERGY_BODY2;
-      });
-
-      RAJA::forall<RAJA::omp_target_parallel_for_exec<NUMTEAMS>>(
-        RAJA::RangeSegment(ibegin, iend), [=](int i) {
-        ENERGY_BODY3;
-      });
-
-      RAJA::forall<RAJA::omp_target_parallel_for_exec<NUMTEAMS>>(
-        RAJA::RangeSegment(ibegin, iend), [=](int i) {
-        ENERGY_BODY4;
-      });
-
-      RAJA::forall<RAJA::omp_target_parallel_for_exec<NUMTEAMS>>(
-        RAJA::RangeSegment(ibegin, iend), [=](int i) {
-        ENERGY_BODY5;
-      });
-
-      RAJA::forall<RAJA::omp_target_parallel_for_exec<NUMTEAMS>>(
-        RAJA::RangeSegment(ibegin, iend), [=](int i) {
-        ENERGY_BODY6;
-     });
+      RAJA::region<RAJA::seq_region>( [=]() {
+
+        RAJA::forall<RAJA::omp_target_parallel_for_exec<NUMTEAMS>>(
```

Comment:
Not sure if this matters, but I merged https://github.com/LLNL/RAJA/pull/539 not long ago, changing "NUMTEAMS" to "ThreadsPerTeam".

Reply:
Yes,  I'll update to a newer develop branch and change those for the release.